git_pull/DOCS.md

```diff
@@ -8,6 +8,13 @@ Follow these steps to get the add-on installed on your system:
 2. Find the "Git pull" add-on and click it.
 3. Click on the "INSTALL" button.
 
+## WARNING
```

Comment:
Does this need to be capatilized?

Reply:
Good point, no.  I just wanted something to get the user's attention.